AUTHORS

```diff
@@ -10,4 +10,4 @@
 
 # Please keep the list sorted.
 
-Vastech SA (PTY) LTD
+GoGraphviz Authors
```

Comment:
No this should be Vastech, since it has copyright.
GoGraphviz authors is shorthand for the list of authors contained in this file